packages/next/src/build/utils.ts

```diff
@@ -1336,7 +1336,7 @@ export async function buildAppStaticPaths({
             store.fetchCache = current.config.fetchCache
           }
           if (typeof current.config?.revalidate !== 'undefined') {
-            store.revalidate = current.config.revalidate
+            // TODO: Is this needed for anything?
```

Comment:
I don't believe this or any of these store sets are consumed anywhere higher because `generateStaticParams` is just during build/dev and then the store is tossed.